src/Hyperparameters.jl

```diff
@@ -17,8 +17,15 @@ Collection of all the hyperparameters, and their values accessed during this run
 const HYPERPARAMETERS = Dict{Symbol, Any}()
 const SAGEMAKER_PREFIX = "SM_HP_"
 
+
+_name_to_envvar(prefix, name) = uppercase(string(prefix, name))
+function _envvar_to_name(prefix, name)
+    startswith(name, prefix) || throw(DomainError(name, "should have prefix; $prefix"))
```

Comment:
```suggestion
    startswith(name, prefix) || throw(DomainError(name, "Should have prefix; $prefix"))
```

Reply:
Also do we test this error?

Reply:
we don't.
Though, it's an internal error, it can't happen unless you call this directly.
It could be an `@assert`?